rest-api-spec/src/main/resources/rest-api-spec/api/indices.get.json

```diff
@@ -4,17 +4,12 @@
     "methods":[ "GET" ],
     "url":{
       "path":"/{index}",
-      "paths":[ "/{index}", "/{index}/{feature}" ],
```

Comment:
Should we also add rest specs and yaml tests for the new REST endpoints introduced in this PR?